test/unit/model/count.test.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+/* jshint -W030 */
+var chai = require('chai')
+  , expect = chai.expect
+  , Support = require(__dirname + '/../support')
+  , current = Support.sequelize
+  , sinon = require('sinon')
+  , DataTypes = require(__dirname + '/../../../lib/data-types')
+  , Promise = require('bluebird');
+
+describe(Support.getTestDialectTeaser('Model'), () => {
+  describe.only('method count', () => {
+    before(() => {
+      this.oldFindAll = current.Model.findAll;
```

Comment:
You can preface with `sandbox = sinon.sandbox.create(), and use `sandbox.stub`. Then you just need to call `sandbox.restore()` in  ` afterEach` to reset all the stubs